### PR TITLE
feat: add identity LOC request.

### PR DESCRIPTION
**Pull request by benoitdevos**
* feat: add identity LOC request.
* chore: improve LOC doc.

logion-network/logion-internal#574

### CODE REVIEW

**Review by gdethier — APPROVED** [review]
Good job. I left one suggestion and one question.

File: website/docs/client/img/identity-loc-state.puml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@startuml
+hide empty description
+[*] --> PendingRequest: requestIdentityLoc()
```
[the remark this answers is not in the document]
Comment (benoitdevos) [reply website/docs/client/img/identity-loc-state.puml:3]: Yes. Baby steps, I suspect our PO will slightly deviate from standard LOC behavior .

File: packages/client/src/Loc.ts [hunk 2]
```diff
@@ -139,15 +140,23 @@ export class LocsState {
         });
     }
 
-    async requestLoc(params: CreateLocRequestParams & { locType: DataLocType }): Promise<PendingRequest> {
-        const { legalOfficer, locType, description, userIdentity } = params;
+    async requestIdentityLoc(params: CreateLocRequestParams): Promise<PendingRequest> {
```
[the remark this answers is not in the document]
Comment (benoitdevos) [reply packages/client/src/Loc.ts:143]: Good point. Fixed.

File: website/docs/client/img/identity-loc-state.puml (hunk 1 above)
[the remark this answers is not in the document]
Comment (benoitdevos) [reply website/docs/client/img/identity-loc-state.puml:3]: Yes, of course.

**Pull request merged**